rts/Game/UI/KeyBindings.cpp

```diff
@@ -305,6 +305,25 @@ void CKeyBindings::Kill()
 
 
 /******************************************************************************/
+const CKeyBindings::ActionList& CKeyBindings::GetActionList(const std::string& matcher) const
+{
+	static ActionList merged; //FIXME switch to thread_local (?)
+	const ActionList* alPtr;
+
+	for (const auto& p: bindings) {
+		const ActionList& al = p.second;
+
+		if (matcher.empty()) {
```

Comment:
I wonder if there's a way we can make this more efficient? Though this seems to no be critical perf code.